### PR TITLE
example_problem.py

### PR DESCRIPTION
**Pull request by amymhaddad**

### CODE REVIEW

File: main_example_problem.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# inputs for function
+    bill_amount = float(input("What is the bill? "))
+    tip_percent = float(input("What is the tip percent? "))
+
+# call function with variables
+    tip_calculation = tip_calculator(bill_amount, tip_percent)
+
+# coordinate output
+    bill_with_tip = bill_amount + tip_calculation
```
Comment (paulghaddad) [inline main_example_problem.py]: This is something that can also be moved into a function `calculate_total_bill`, passing in the `bill_amount` and `tip`.

File: main_example_problem.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
```
Comment (paulghaddad) [inline main_example_problem.py]: Excellent job organizing everything in this file!

File: test_example_problem.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import unittest
+from example_problem import tip_calculator
+
+class TipTestCase(unittest.TestCase):
+    """Test for calculating tips at restaurants"""
+
+    def test_tip_calculator(self):
+        """Test to see if calculator returns the correct value"""
+
+        bill = 100
```
Comment (paulghaddad) [inline test_example_problem.py]: Awesome job here!!! One thing I'd recommend getting in the habit of is separating the phases of the tests with blank lines:

```
# setup
bill = 100
tip_rate = 20

# exercise code
total_tip = tip_calculator(bill, tip_rate)

# verify the results
self.assertEqual(total_tip, 20)
```

See this video for a good run-down: https://thoughtbot.com/upcase/videos/four-phases-of-testing

File: test_example_problem.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        tip_rate = 20
+        total_tip = tip_calculator(bill, tip_rate)
+        self.assertEqual(total_tip, 20)
```
Comment (paulghaddad) [inline test_example_problem.py]: This test looks great!

**Comment by amymhaddad** [comment]
@paulghaddad I made updates to all of these files!

**Pull request merged**